libs/branding/src/lib/logo/logo.component.scss

```diff
@@ -0,0 +1,39 @@
+$logo-daff-width: 48%;
+$logo-flower-width: 15%;
+$logo-dil-width: 31%;
+$logo-spacer-width: 3%;
```

Comment:
4%

Reply:
@xelaint I tried to approximate as best as I could in photoshop from the svg. These have to sum to 100%. Formula is:

```
$daff + $flower + $dil + 2*$spacer;
```

Reply:
Ah okay, I'll play around with it.